software/spree-commerce.yml

```diff
@@ -8,7 +8,7 @@ platforms:
 tags:
   - E-commerce
 source_code_url: https://github.com/spree/spree
-demo_url: https://new-ux.spreecommerce.org/
-stargazers_count: 12783
+demo_url: https://demo.spreecommerce.org/
+stargazers_count: 12782
```

Comment:
Please don't manually update the stargazers_count. Our bot updates it automatically once per day.